Dockerfile

```diff
@@ -3,9 +3,11 @@ FROM alpine:3.16
 ENV TZ "UTC"
 ENV ENABLE_V6 "no"
 ENV HOSTS "public:localhost"
+ENV USERID "100"
+ENV GROUPID "101"
 
```

Comment:
@TweakM Can you please, just set the defaults to a higher ID? Since being below 500, it may clash with the host OS managed.

I, personally, would go with 1000 or 999. :)

Reply:
Or are these IDs created automatically by lighttpd?

Reply:
@fboaventura I agree that one should normally go for 1000 for the IDs. To prevent breaking existing running containers when they are upgraded I kept the values that the current setup has by default:
/etc/passwd:
lighttpd:x:100:101:lighttpd:/var/www/localhost/htdocs:/sbin/nologin
New users can safely use the values they need. Do you agree?